### PR TITLE
Added a new type of dialog with yes/no/cancel options

### PR DESCRIPTION
**Pull request by easythrees**

### CODE REVIEW

File: conductor/lib/pyside_utils.py [hunk 1]
```diff
@@ -217,6 +217,76 @@ def launch_yes_no_dialog(title, message, show_not_again_checkbox=True, parent=No
...
+
+    dialog.widget = QtWidgets.QWidget(dialog)
+    dialog.horizontalLayout = QtWidgets.QHBoxLayout(dialog.widget)
+    dialog.horizontalLayout.setContentsMargins(-1, -1, -1, 0)
+    dialog.horizontalLayout.setObjectName("horizontalLayout")
+    dialog.verticalLayout.addWidget(dialog.widget)
+
+    # Create the "Don\t ask again" checkbox
+    dialog.checkBox = QtWidgets.QCheckBox(dialog.widget)
+    dialog.checkBox.setText("Don\'t ask again")
```
Comment (flebel) [inline conductor/lib/pyside_utils.py]: You shouldn't have to escape the single quote if it's in a double quotes string.

File: conductor/lib/pyside_utils.py [hunk 2]
```diff
@@ -217,6 +217,76 @@ def launch_yes_no_dialog(title, message, show_not_again_checkbox=True, parent=No
...
+    dialog.label.setText(message)
+    dialog.verticalLayout.addWidget(dialog.label)
+
+    dialog.widget = QtWidgets.QWidget(dialog)
+    dialog.horizontalLayout = QtWidgets.QHBoxLayout(dialog.widget)
+    dialog.horizontalLayout.setContentsMargins(-1, -1, -1, 0)
+    dialog.horizontalLayout.setObjectName("horizontalLayout")
+    dialog.verticalLayout.addWidget(dialog.widget)
+
+    # Create the "Don\t ask again" checkbox
```
Comment (flebel) [inline conductor/lib/pyside_utils.py]: `s/Don\\t/Don't`/

File: conductor/lib/pyside_utils.py [hunk 3]
```diff
@@ -217,6 +217,76 @@ def launch_yes_no_dialog(title, message, show_not_again_checkbox=True, parent=No
...
+    dialog = QtWidgets.QDialog(parent=parent)
+    dialog.setWindowTitle(title)
+    dialog.verticalLayout = QtWidgets.QVBoxLayout(dialog)
+
+    # Create the dialogs main message (Qlabel)
+    dialog.label = QtWidgets.QLabel(dialog)
+    dialog.label.setAlignment(QtCore.Qt.AlignCenter)
+    dialog.label.setTextInteractionFlags(dialog.label.textInteractionFlags() | QtCore.Qt.TextBrowserInteraction)
+    dialog.label.setTextFormat(QtCore.Qt.RichText)
+    dialog.label.setOpenExternalLinks(True)
```
Comment (WMDConductor) [inline conductor/lib/pyside_utils.py]: Why would this need to be set to True?

File: conductor/lib/pyside_utils.py [hunk 4]
```diff
@@ -217,6 +217,76 @@ def launch_yes_no_dialog(title, message, show_not_again_checkbox=True, parent=No
...
+    dialog.label.setText(message)
+    dialog.verticalLayout.addWidget(dialog.label)
+
+    dialog.widget = QtWidgets.QWidget(dialog)
+    dialog.horizontalLayout = QtWidgets.QHBoxLayout(dialog.widget)
+    dialog.horizontalLayout.setContentsMargins(-1, -1, -1, 0)
+    dialog.horizontalLayout.setObjectName("horizontalLayout")
```
Comment (WMDConductor) [inline conductor/lib/pyside_utils.py]: Is there a need to explicitly set the layout name here?

Comment (easythrees) [reply conductor/lib/pyside_utils.py]: I used the layout from the other one, my assumption was that it was needed.

File: conductor/lib/pyside_utils.py (hunk 3 above)
Comment (easythrees) [reply conductor/lib/pyside_utils.py]: I just copied it from the other one, but I can change it when I implement the Nuke version.

Comment (easythrees) [reply conductor/lib/pyside_utils.py]: removed this line.

File: conductor/lib/pyside_utils.py (hunk 4 above)
Comment (easythrees) [reply conductor/lib/pyside_utils.py]: removed this line

File: conductor/lib/pyside_utils.py (hunk 1 above)
Comment (easythrees) [reply conductor/lib/pyside_utils.py]: Well I'll be, it works! Changed.

File: conductor/lib/pyside_utils.py (hunk 2 above)
Comment (easythrees) [reply conductor/lib/pyside_utils.py]: Changed

**Pull request merged**